source/module_io/to_wannier90.cpp

```diff
@@ -798,7 +803,7 @@ void toWannier90::produce_trial_in_pw(const int &ik, ModulePW::PW_Basis_K *wfc_b
         }
     }
 
-    const int npw = GlobalC::kv.ngk[ik];
+    const int npw = wfc_pw.get_ngk(ik);
```

Comment:
In fact, we do not need wfc_pw, we can use wfc_basis->ngk[ik]

Reply:
I cannot find ngk in both class PW_Basis and PW_Basis_k, can you?

Reply:
npwk[ik] in PW_Basis_k

Reply:
OK, thanks!